src/backend/cuda/jit/kernel_generators.hpp

```diff
@@ -71,8 +71,9 @@ void generateBufferRead(std::stringstream& kerStream, int id,
               << "];\n";
 }
 
-void generateShiftNodeOffsets(std::stringstream& kerStream, int id,
-                              bool is_linear, const std::string& type_str) {
+inline void generateShiftNodeOffsets(std::stringstream& kerStream, int id,
```

Comment:
nit: looks like this is just to make the compiler silent even though this won't be actually inlined.